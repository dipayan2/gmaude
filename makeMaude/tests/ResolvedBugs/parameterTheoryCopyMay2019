#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/parameterTheoryCopyMay2019.maude -no-banner -no-advise \
  > parameterTheoryCopyMay2019.out 2>&1

diff $srcdir/parameterTheoryCopyMay2019.expected parameterTheoryCopyMay2019.out > /dev/null 2>&1
