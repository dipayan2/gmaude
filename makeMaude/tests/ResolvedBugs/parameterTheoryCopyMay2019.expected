Warning: <automatic>: sort Z$Elt has been imported from both <standard input>,
    line 12 (fth TRIV') and <standard input>, line 7 (fmod FOO). Since it is
    imported from both a module and a theory, this renders theory Z :: TRIV'
    unusable.
Warning: <standard input>, line 19 (fmod BAZ): this module contains one or more
    errors that could not be patched up and thus it cannot be used or imported.
Warning: <standard input>, line 19 (fmod BAZ): module BAZ is unusable due to
    unpatchable errors.
Bye.
